WordPress/Classes/ViewRelated/Gutenberg/Utils/GutenbergStockPhotos.swift

```diff
@@ -33,30 +36,40 @@ extension GutenbergStockPhotos: StockPhotosPickerDelegate {
             return
         }
 
+        // For blocks that support multiple uploads this will upload all images.
+        // If multiple uploads are not supported then it will seperate them out to Image Blocks.
+        multipleSelection ? insertOnBlock(with: assets) : insertSingleImages(assets)
+    }
+
+    /// Adds the given image object to the requesting block and seperates multiple images to seperate image blocks
+    /// - Parameter asset: Stock Media object to add.
+    func insertSingleImages(_ assets: [StockPhotosMedia]) {
         // Append the first item via callback given by Gutenberg.
         if let firstItem = assets.first {
-            insertOnBlock(with: firstItem)
+            insertOnBlock(with: [firstItem])
         }
         // Append the rest of images via `.appendMedia` event.
         // Ideally we would send all picked images via the given callback, but that seems to not be possible yet.
```

Comment:
np: although not part of the changeset, should this comment be updated (i.e. does this PR make this possible now)?

Reply:
Good question! When I tested it out by forcing the Image Block down the multiple upload path with all of the images, it would only upload the first image and drop the rest. With the Gallery Block enabled now, it might be worth revisiting how to handle this case.